### PR TITLE
Change name to @maplibre/maplibre-gl-leaflet in package.json

### PR DESCRIPTION
**Pull request by astridx**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "maplibre-gl-compare",
+  "name": "@maplibre/maplibre-gl-compare",
   "version": "0.5.0",
   "description": "Swipe and sync between two Maplibre maps",
```
Comment (wipfli) [inline package.json]: Please change `Maplibre` to `MapLibre`

Comment (astridx) [reply package.json]: Which Maplibre exactly? 

Shouldn't that be consistent? Here is also in lower case: https://github.com/maplibre/maplibre-gl-leaflet/blob/main/package.json

Comment (wipfli) [reply package.json]: Sorry for not being clear. I am referencing the `description`, swipe an sync between.... The `name` is fine as it is now.

**Comment by astridx** [comment]
Done. Thanks for the hint.

Would you please help me further? What do I need to do now to publish on npm?

**Comment by wipfli** [comment]
You can run the workflow manually again and if all looks fine, remove the `--dry-run` flag and run again. Then it should be published.

**Comment by astridx** [comment]
> You can run the workflow manually again and if all looks fine, remove the `--dry-run` flag and run again. Then it should be published.

Before I have to merge this, right?

**Pull request merged**